packages/components/date-picker/src/date-picker-com/basic-date-table.vue

```diff
@@ -4,8 +4,7 @@
     :aria-label="t('el.datepicker.dateTablePrompt')"
     cellspacing="0"
     cellpadding="0"
-    class="el-date-table"
-    :class="{ 'is-week-mode': selectionMode === 'week' }"
+    :class="[ns.b(), { 'is-week-mode': selectionMode === 'week' }]"
```

Comment:
```suggestion
    :class="[ns.b(), { ns.is('week-mode'): selectionMode === 'week' }]"
```

Reply:
`{ ns.is('week-mode'): selectionMode === 'week' }` We can't seem to use functions as object properties.